client/FileUpload.js

```diff
@@ -39,12 +39,17 @@ class FileUpload extends Component {
         super(props);
         this.state = {
             fileDoc: undefined,
+            isBioPhoto: (this.props.bio && this.props.bio === "true") ? true : false
```

Comment:
no need to add this to state, just use `this.props.bio` in place of `this.state.bio`